solution/ui/regulations/eregs-vite/src/views/Subjects.vue

```diff
@@ -200,18 +210,30 @@ const policyDocList = ref({
     error: false,
 });
 
-const getDocList = async (requestParamsString = "") => {
+const getDocList = async ({ requestParamString = "", query, type }) => {
     policyDocList.value.loading = true;
     policyDocList.value.error = false;
 
-    const requestParams = `${requestParamsString}&page_size=${pageSize}&paginate=true`;
+    const requestParams = `${requestParamString}&page_size=${pageSize}`;
+    const docType = type ? DOCUMENT_TYPES_MAP[type] : undefined;
+
+    let contentList;
 
     try {
-        const contentList = await getCombinedContent({
-            apiUrl: props.apiUrl,
-            cacheResponse: false,
-            requestParams,
-        });
+        if (query) {
+            contentList = await getCombinedContent({
+                apiUrl: props.apiUrl,
+                requestParams,
+                docType,
+            });
+        } else {
+            contentList = await getContentWithoutQuery({
+                apiUrl: props.apiUrl,
+                requestParams,
+                docType,
+            });
+        }
```

Comment:
Since `content-search` requires a query string in order to return results, we have to conditionally use a different endpoint if no query string is present (`/resources` if no doc type is specified; `/resources/public` if only Public documents are requested; `resources/internal` if only Internal documents are requested

Ex: user searches for "FMAP" in category "3".  A request is made to `/content-search`
Ex: user searches for category "3" only.  A request is made to `/resources`